src/Hyperwallet/Response/ErrorResponse.php

```diff
@@ -33,6 +40,9 @@ class ErrorResponse implements \Countable, \ArrayAccess {
     public function __construct($statusCode, array $errors) {
         $this->statusCode = $statusCode;
         $this->errors = array_map(function ($error) {
+            if (!isset($relatedResources) && isset($error['relatedResources'])) {
```

Comment:
why do we need this extra code? `!isset($relatedResources)` this is from constructor then for sure this is always not set, isn't it?

Reply:
Actually we need this check !isset($relatedResources)
array_map applies that function for each element of array in $errors['errors'] and we have to set relatedResources only in case we haven't done it already (!isset($relatedResources) )